src/components/navbar.component.tsx

```diff
@@ -55,6 +61,43 @@ const Navbar: React.FC<NavbarProps> = ({ user, onLogout, allowedLocales }) => {
     };
   }, []);
 
+  let [searchBarIsVisible, setSearchBarIsVisible] = React.useState<any>(false);
+
+  let [closeSearchBar, setCloseSearchBar] = React.useState<any>(false);
+
+  const setSearchBar = () => {
+    setSearchBarIsVisible((searchBarIsVisible = true));
```

Comment:
```suggestion
    setSearchBarIsVisible(true);
```

Reply:
Ooh i see , thanks alot @donaldkibet  , i appreciate this